src/components/SidebarTabs/SidebarTabs.js

```diff
@@ -1,43 +1,14 @@
-import { observer } from 'mobx-react';
-import React, { createContext, useState } from 'react';
-import { Children } from 'react';
+import React from 'react';
 import { Block, Elem } from '../../utils/bem';
 import './SidebarTabs.styl';
 
-const SidebarContext = createContext();
-
-export const SidebarTabs = observer(({ active, children }) => {
-  const [selected, setSelected] = useState(active);
-  const tabs = Children.toArray(children);
-
+// @todo there was an idea of switchable tabs, but they were not used,
+// @todo so implementation was removed and the whole part of interface
+// @todo is waiting to be removed in favor of new UI (see FF_DEV_3873)
```

Comment:
I see this FF is on for all but 5 users on production - what do we need to do to get buy in for the remaining 5 so we can archive this FF and remove the outdated code? this'll help clean up this portion of the code